src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -401,7 +401,6 @@ public static class Agent
                 public static readonly string Version = "agent.version";
                 public static readonly string WorkFolder = "agent.workfolder";
                 public static readonly string WorkingDirectory = "agent.WorkingDirectory";
-                public static readonly string EnableAdditionalMaskingRegexes = "agent.enableadditionalmaskingregexes";
```

Comment:
After a lot of confusion, I think n that the new flag does not need to be included in this file. At one time, additional regexes were turned on via a variable, but now they are turned on strictly using the environment variable (even before this change). I believe this might be due to needing to replace `new SecretMasker()` that needs to be done very early before tracing can be set-up. Both this change and the change prior to this one that wired a flag to using a built-in secret masker have this requirement, so I think this could have been removed already, and I think it is correct to remove it now.